test/jdk/javax/swing/JButton/HtmlButtonImageTest/HtmlButtonImageTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+*/
+
+/*
+ * @test
+ * @bug 8015854
+ * @requires (os.family == "mac")
+ * @summary Tests HTML image as JButton text for unwanted padding on macOS Aqua LAF
+ * @run main HtmlButtonImageTest
+ */
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.awt.image.BufferedImage;
+import javax.swing.JButton;
+import javax.swing.UIManager;
+import javax.swing.SwingUtilities;
+import java.nio.file.Path;
+import java.io.IOException;
+import java.io.File;
+import javax.imageio.ImageIO;
```

Comment:
Could you sort the imports, please? *Optimize Imports*  or a similar command in your IDE should do the magic.